inst/qml/Prophet.qml

```diff
@@ -0,0 +1,626 @@
+import QtQuick 			2.8
+import QtQuick.Layouts 	1.3
+import JASP.Controls 	1.0
+import JASP.Widgets 	1.0
+import JASP				1.0
+
+Form
+{
+	VariablesForm
+	{	
+		preferredHeight: 400 * preferencesModel.uiScale
+		AvailableVariablesList	{ name: "allVariablesList" }
+		AssignedVariablesList	{ name: "dependent"; title: qsTr("Dependent Variable");	suggestedColumns: ["scale"]; singleVariable: true													}
+		AssignedVariablesList	{ name: "time"; title: qsTr("Time"); suggestedColumns: ["nominal"]; singleVariable: true																	}
```

Comment:
You also managed quite well handling this, nice! I think in the future we could also think about implementing a time/date type column, that would help a lot, too. At the moment it still feels a bit wonky and puts a lot of requirements on the user to format the time/date column exactly how you expect. @JorisGoosen do you have an idea how hard/problematic/stupid would be implementing time/date as a type of column in the JASP data viewer?

Reply:
https://github.com/jasp-stats/INTERNAL-jasp/issues/312 already exists.
I do not think it is particularly hard but till now nobody really seemed to need it.

As in, some people complained that the date-time in JASP looks completely messed up when loading from SPSS or so. So I guess they would be happy.

If this would be useful for this module I could see about adding this in the nearfuture, shouldn't be so bad.
But it would good to know what kind of data R then expects. Maybe either of you could add a description of how you/prophet likes to get its dates formatted. That way I can make sure it is delivered in the right format.

Reply:
Yes, indeed, Prophet can only handle nominal dates/times from JASP when they're in ISO format (which is R's default). That is they need to be in the format "YYYY-mm-dd hh:mm:ss" where the ":ss" or "hh:mm:ss" can be omitted and "-" can be replaced by "/". But I think I could change the conversion so that any other uniquely identifiable format is the default (but it would need to be the only format to convert).

But in general, having a date/time type in JASP would give the module more flexibility!

Reply:
I copied your remarks to that issue, because otherwise I need to keep following links all through github when I get time to work on it ;)

Reply:
Oh yes, thanks!